plugin/trino-iceberg/pom.xml

```diff
@@ -402,9 +408,21 @@
                                 <include>**/TestIcebergGlueCatalogConnectorSmokeTest.java</include>
                                 <include>**/TestTrinoGlueCatalogTest.java</include>
                                 <include>**/TestSharedGlueMetastore.java</include>
+                                <include>**/TestIcebergGlueCatalogAccessOperations.java</include>
                             </includes>
                         </configuration>
                     </plugin>
+                    <plugin>
```

Comment:
Copied from above, was getting error for this when running the iceberg glue profile so I copied into the profile

Reply:
The above shouldn't be in `default` profile.

Reply:
https://github.com/trinodb/trino/pull/11540